docs/user-guide/how_to_use_vgpu.md

```diff
@@ -193,3 +193,51 @@ Tue Mar  7 12:19:59 2023
 [4pdvGPU Msg(31336:140369108928320:multiprocess_memory_limit.c:457)]: Calling exit handler 31336
 ...
 ```
+
+### VGPU Monitor
+
+VGPU related monitor metrics can be found in volcano scheduler metrics:
+
+{scheduler ip}:8080/metrics
+
+### Schedule Policy
+
+You need to actiave nodeorder plugin and set nodeorder.devicescore.weight > 0 to enable schedule policy, see [how_to_configure_scheduler](./how_to_configure_scheduler.md) for more information
+
+Then you can set one of the following schedule policy using predicate argument "predicate.VgpuSchedulePolicy"
+
+There are three schedule policies, as the following table:
+
+| predicate.VgpuSchedulePolicy      | Description |
+| ----------- | ----------- |
+| 0      | No schedule policy       |
```

Comment:
Why number here instead of a specific policy name?

Reply:
because we can only assign "float","bool","int" in arguments

Reply:
This seems user unfriendly.

Reply:
agree, but right now args.GetString haven't been implemented yet, so i can only pass an "int" to argument